lib/experimental/posts/load.php

```diff
@@ -7,22 +7,95 @@
 
 add_action( 'admin_menu', 'gutenberg_replace_posts_dashboard' );
 
+// Default to is-fullscreen-mode to avoid jumps in the UI.
+add_filter(
+	'admin_body_class',
+	static function ( $classes ) {
+		return "$classes is-fullscreen-mode";
+	}
+);
+
 /**
  * Renders the new posts dashboard page.
  */
 function gutenberg_posts_dashboard() {
+	// Flag that we're loading the block editor.
+	// $current_screen = get_current_screen();
```

Comment:
Are these code comments still required?

Reply:
Still need to do at least a first good pass of needed settings. I'll update soon.